docs/developer-experience/recipes/client-app-inner-loop.md

```diff
@@ -0,0 +1,165 @@
+# Strategies for a High Productivity Inner Dev Loop when developing Client Apps

```

Comment:
The content looks great, but I think the title is misleading.  The same could be applied for any application component that has remote dependencies, not just client apps.